app/pages/notifications/message.cjsx

```diff
@@ -0,0 +1,51 @@
+React = require 'react'
+{Link} = require '@edpaget/react-router'
+{Markdown} = require 'markdownz'
+moment = require 'moment'
+talkClient = require '../../api/talk'
+Loading = require '../../components/loading-indicator'
+Avatar = require '../../partials/avatar'
+
+module?.exports = React.createClass
+  displayName: 'MessageNotification'
+
+  propTypes:
+    project: React.PropTypes.object
+    user: React.PropTypes.object.isRequired
+    notification: React.PropTypes.object.isRequired
+
+  getInitialState: ->
+    message: null
+    messageUser: null
+    conversation: null
+
+  componentWillMount: ->
+    talkClient.type('messages').get(@props.notification.source_id, include: 'conversation,user').then (message) =>
+      message.get('user').then (messageUser) =>
+        message.get('conversation').then (conversation) =>
+          @setState {message, conversation, messageUser}
```

Comment:
This is actually only one API call since the user and conversation is sideloaded.

If there's an easier way to do this with the client, I'd be happy to change it